File: MPI/mpicheck/tests/functions/4.0/MPI_Pready_range.f

        
        program main
        include 'mpif.h'
        
        INTEGER var_0
       INTEGER var_1
       INTEGER var_2
       INTEGER var_3
        call mpi_pready_range(var_0, var_1, var_2, var_3)
       call pmpi_pready_range(var_0, var_1, var_2, var_3)
        end program main
    

File: MPI/mpicheck/tests/functions/4.0/MPI_Pready_range.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    int var_0;
    int var_1;
    MPI_Request var_2;
    int ret;
    /* calls */
    ret = MPI_Pready_range(var_0, var_1, var_2);
    ret = PMPI_Pready_range(var_0, var_1, var_2);
    return 0;
}
